tephra-core/src/main/java/org/apache/tephra/distributed/TransactionServiceClient.java

```diff
@@ -130,10 +134,11 @@ public static void doMain(boolean verbose, Configuration conf) throws Exception
    * for service discovery. Otherwise it will look for the port in the
    * config and use localhost.
    * @param config a configuration containing the zookeeper properties
+   * @param clientId id of the client that identifies it when it starts a transaction
    */
   @Inject
-  public TransactionServiceClient(Configuration config,
-                                  ThriftClientProvider clientProvider) {
+  public TransactionServiceClient(Configuration config, ThriftClientProvider clientProvider,
```

Comment:
@poornachandra Addressed comment in the commit https://github.com/apache/incubator-tephra/pull/42/commits/0279658b3b0718298ccfe363343bce182e4ba655. Please take a look again.